src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -1544,7 +1518,23 @@ public async IAsyncEnumerable<SourceFile> Load(SessionId id, string[] loaded_fil
                 }
                 catch (Exception e)
                 {
-                    logger.LogError($"Failed to load {step.Url} ({e.Message})");
+                    try
+                    {
+                        if (tryUseDebuggerProtocol)
+                        {
+                            string unescapedFileName = Uri.UnescapeDataString(step.Url);
+                            bytes = await context.SdbAgent.GetBytesFromAssemblyAndPdb(Path.GetFileName(unescapedFileName), token);
+                            assembly = new AssemblyInfo(monoProxy, id, step.Url, bytes[0], bytes[1], logger, token);
+                        }
+                        else
+                        {
+                            logger.LogDebug($"Failed to read {step.Url} ({e.Message})");
```

Comment:
I think he meant https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.logerror?view=dotnet-plat-ext-7.0#microsoft-extensions-logging-loggerextensions-logerror(microsoft-extensions-logging-ilogger-system-exception-system-string-system-object()) . Does the error show up in VS? how?